components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -159,6 +161,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
         [aDecoder decodeObjectForKey:MDCTextInputFundamentCharacterRelativeSuperviewKey];
     _textColor = [aDecoder decodeObjectForKey:MDCTextInputFundamentTextColorKey];
     _trailingUnderlineLabel = [aDecoder decodeObjectForKey:MDCTextInputFundamentTrailingLabelKey];
+    _trailingViewMode = (UITextFieldViewMode)[aDecoder decodeIntegerForKey:MDCTextInputFundamentTrailingViewKey];
```

Comment:
Spoke with you offline. I do indeed need these checks. Let's do that in another PR for there are more than just this one.